share/mk/bsd.compat.mk

```diff
@@ -157,9 +157,13 @@ LIB64_MACHINE_ABI=	${MACHINE_ABI:Npurecap}
 .endif
 HAS_COMPAT+=CHERI
 LIBCHERICPUFLAGS=  -target mips64-unknown-freebsd13.0 -cheri -mabi=purecap
+LIBCHERICPUFLAGS+=	-fpic
+LIBCHERICPUFLAGS+=	-Qunused-arguments
```

Comment:
Is this still needed? I feel like I fixed most flags to be marked as consumed in clang.

Reply:
I do still get warnings, but they aren't fatal.  They scrolled off my terminal, but I was seeing warnings about unused "-L" passed to clang-cpp during the libcheri build of a hybrid world.  I will keep this removed though if my world finishes without errors.